src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,34 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+
+export const PostInfo = ({ post }) => {
+  const { title, body, user, comments } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        <p>
+          {' Posted by  '}
+
+          {user && <UserInfo user={user} />}
+        </p>
+      </div>
+
+      <p className="PostInfo__body">{body}</p>
+
+      <hr />
+
+      {comments.length ? (
+        <CommentList comments={comments} />
+      ) : (
+        <>
+          <hr />
```

Comment:
Two `hr`s are rendered when there are no comments, only one is needed:

![image](https://github.com/mate-academy/react_static-list-of-posts-js/assets/52002856/3d548c8c-d492-4dac-8c5f-3c7482b373a2)